hackage-security-http-client/src/Hackage/Security/Client/Repository/HttpLib/HttpClient.hs

```diff
@@ -80,10 +84,8 @@ getRange manager reqHeaders uri (from, to) callback = wrapCustomEx $ do
          () | HttpClient.responseStatus response == HttpClient.ok200 ->
            callback HttpStatus200OK (getResponseHeaders response) br
          _otherwise ->
-           throwChecked $ HttpClient.StatusCodeException
-                            (HttpClient.responseStatus    response)
-                            (HttpClient.responseHeaders   response)
-                            (HttpClient.responseCookieJar response)
+           throwChecked $ HttpClient.HttpExceptionRequest request
```

Comment:
I don't think so; the ultimate value passed to `withResponse` is in fact the non-ticked version, `request`.